Assignments/A3/Distro/QLearn.c

```diff
@@ -331,13 +351,14 @@ void maxQsa(double gr[max_graph_size][4], double weights[25],
   int size_Y = graph_size / size_X;
   int next_mouse_pos[1][2];
   for (int action = 0; action < numActions; ++action) {
-    double features[25];
+    double features[numFeatures];
```

Comment:
Since `evaluateFeatures` expects a double array of size 25, should be `features[25];`
(`evaluateFeatures` internally uses `numFeatures` to only fill in the necessary ones).